src/inc/dacprivate.h

```diff
@@ -20,6 +20,14 @@
 #include <msodw.h>
 #endif // FEATURE_PAL
 
+#if !defined(MSLAYOUT)
```

Comment:
I did a little experiment - putting the definition only to dacprivate.h and adding `#include "dacprivate.h"` to src/debug/inc/common.h seems to work.